dashboard/db/migrate/20160517173415_add_users_to_unit_and_module_assignments.rb

```diff
@@ -0,0 +1,16 @@
+class AddUsersToUnitAndModuleAssignments < ActiveRecord::Migration
+  def change
+    add_reference :plc_enrollment_unit_assignments, :user, index: true
+    add_reference :plc_enrollment_module_assignments, :user, index: true
```

Comment:
I've been avoiding them for enrollment stuff, and really most plc stuff. Trying to remember why